vital/tests/test_image.cxx

```diff
@@ -36,6 +37,33 @@ value_at( unsigned int i, unsigned int j, unsigned int k )
   return static_cast<byte>( ( ( ks * k ) + ( js * j ) + ( is * i ) ) % 255 );
 }
 
+// ----------------------------------------------------------------------------
+void check_equal( image_memory mem_1, image_memory mem_2 )
```

Comment:
You probably should pass by `const &`

Reply:
The most recent commit addresses this, but I needed to make the `data()` function const, which included changing it in the ocv and vxl arrows. Let me know if that wasn't the right way to do things

Reply:
These changes were determined to be out of the scope of this PR, so for now we're just leaving this as is. (that "most recent commit" I referenced has been removed)